third_party/terraform/tests/resource_compute_backend_service_test.go.erb

```diff
@@ -752,6 +752,132 @@ func TestAccComputeBackendService_internalLoadBalancing(t *testing.T) {
 }
 <% end -%>
 
+<% unless version == 'ga' -%>
+func TestAccComputeBackendService_trafficDirectorUpdate(t *testing.T) {
```

Comment:
Can you add a second update test that updates from `testAccComputeBackendService_trafficDirector` to a config where the localityLbPolicy is one of the simpler ones (and so all the hash stuff is completely unset)? Preferably a different one than the example.

Reply:
I added this and since the API doesn't handle the removal of `consistentHash` very well, I made `localityLbPolicy` input only.